### ISSUE TITLE
Getting _ITERATOR_DEBUG_LEVEL errors during AirSim Windows Build

### ISSUE THREAD

**Issue by IAmSaad** [Microsoft/AirSim#1001]
I am following the Build AirSim Windows guide [here](https://github.com/Microsoft/AirSim/blob/master/docs/build_windows.md) and I get the following errors when building the RPC module.

![ask_github](https://user-images.githubusercontent.com/13678055/39087675-f21dc84e-45bd-11e8-8915-e5e63ed2a320.PNG)

I saw stackoverflow ([this link](https://stackoverflow.com/questions/7668200/error-lnk2038-mismatch-detected-for-iterator-debug-level-value-0-doesnt)) and found some pointers on what may be causing it. This has got something to do with DEBUG and RELEASE versioning. I opened AirSim.sln and manually started adding the preprocessor flags `_HAS_ITERATOR_DEBUGGING=0;_ITERATOR_DEBUG_LEVEL=0;` to AirLib, AirlibUnitUnitTests, DroneServer etc and rebuilding them in Visual Studio but that didn't change the build errors.

**Comment by sytelus** [comment]
You shouldn't be getting any such errors. The build command should work without error. Are you using VS2015 Update 3? Are you doing build command from VS x64 command prompt?

**Comment by IAmSaad** [comment]
Yes I have a fresh install VS2015 Community Update 3 and I am doing it in the VS2015 x64 command prompt. If I use generic command prompt, it doesn't find msbuild.

**Comment by sytelus** [comment]
There was a bug in DroneServer that got just fixed although I don't think its relevant to what you are reporting. For the latest person, I can do build without any errors. You might want to pull latest, make sure you don't have any custom changes and just run build command.

**Comment by IAmSaad** [comment]
Still getting the same error on the latest pull of master. Are the subprojects like DroneServer, Examples, HelloCar, DroneShell etc are being released with preprocessor flag _ITERATOR_DEBUG_VALUE = 0 or debugging turned off? Because all subprojects build fail on my side with this 

> mismatch detected for '_ITERATOR_DEBUG_LEVEL': value '0' doesn't match value
'2' in main.obj

**Comment by sytelus** [comment]
Could you run clean.bat and regenerate solution. I'm not getting this error.

**Comment by IAmSaad** [comment]
Thanks! It builds fine now and I have used VS2017 for the build process now.
